megamek/unittests/megamek/server/totalwarfare/TWDamageManagerTest.java

```diff
@@ -0,0 +1,544 @@
+package megamek.server.totalwarfare;
+
+import megamek.common.*;
+import megamek.common.options.GameOptions;
+import megamek.common.options.IOption;
+import megamek.common.options.Option;
+import megamek.common.options.OptionsConstants;
+import megamek.common.weapons.DamageType;
+import org.junit.jupiter.api.BeforeAll;
```

Comment:
Copyright Header....

Reply:
Renaming the variables will take a while.

Reply:
Actually, can I get some clarification on which variables need renaming?
I don't plan to do any additional work on the code in `TWDamageManager::damageEntity` because it is meant to be as close to the original damage application code as possible.

Reply:
It was items such as `r -> report` kind of stuff. Make the variables more human readable. Things that don't impact the code except make it more readable.

It's just a request, not a requirement.

Reply:
I imagine that future me will appreciate it if we do this now.  :)